setup.cfg

```diff
@@ -0,0 +1,3 @@
+[metadata]
+description-file=README.md
+license_files=LICENSE.rst
```

Comment:
There is an existing license file, called 'LICENSE'.

Reply:
have pointed to LICENSE file as suggested